dc/core/DisplayGroup.cpp

```diff
@@ -179,6 +179,8 @@ DisplayGroup& DisplayGroup::operator=( const DisplayGroup& displayGroup )
 
 void DisplayGroup::clear()
 {
+    put_flog( LOG_INFO, "removing %i windows", contentWindows_.size( ));
```

Comment:
My idea was to have this info in release mode as well, to see when the users clean the screen. I think it help understanding the sequence of events that happen.